includes/Media/Video/Trim.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Class Trim
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Media\Video;
+
+use Google\Web_Stories\Service_Base;
+
+/**
+ * Class Optimization
+ *
+ * @package Google\Web_Stories\Media\Video
+ */
+class Trim extends Service_Base {
```

Comment:
The `Muting` class is called `Muting`. Should we call this `Trimming` accordingly?

Also see the incorrect class name in the docblock here.

Reply:
This is a draft, so I will fix things like docblocks then. 

I changed the class name.